base/_mixins.scss

```diff
@@ -273,7 +283,12 @@
 ///  // @include proportional-container(16 9)
 ///
 /// @example css - Output
-///  // position: relative; overflow: hidden; z-index: 1; height: 0; padding-bottom: 56.25%; width: 100%
+///  // position: relative;
+///  // overflow: hidden;
+///  // z-index: 1;
+///  // height: 0;
+///  // padding-bottom: 56.25%;
+///  // width: 100%
 ///
 @mixin proportional-container( $ratio: 1 1 ) {
```

Comment:
should the spaces inside the ()'s be removed `@mixin proportional-container($ratio: 1 1)`